query/functions_test.go

```diff
@@ -497,3 +503,76 @@ func TestSample_SampleSizeGreaterThanNumPoints(t *testing.T) {
 		t.Fatalf("unexpected points: %s", spew.Sdump(points))
 	}
 }
+
+func TestHll_SumAndMergeHll(t *testing.T) {
+	assert := tassert.New(t)
+	require := trequire.New(t)
+
+	// Make 3000 random strings
+	r := rand.New(rand.NewSource(42))
+	input := make([]*query.StringPoint, 0, 3000)
+	for i := 0; i < 3000; i++ {
+		input = append(input, &query.StringPoint{Value: strconv.FormatUint(r.Uint64(), 10)})
+	}
+
+	// Insert overlapping sections of the same points array to different reducers
+	s1 := query.NewStringSumHllReducer()
+	for _, p := range input[:2000] {
+		s1.AggregateString(p)
+	}
+	point1 := s1.Emit()
+	s2 := query.NewStringSumHllReducer()
+	for _, p := range input[1000:] {
+		s2.AggregateString(p)
+	}
+	point2 := s2.Emit()
+	require.Equal("17190211103962133664", input[2999].Value)
```

Comment:
I'm probably missing the obvious but why is this check of the last input string needed?

Reply:
This is just to show people reading the test the format of the values (they are stringified numbers) and they really are repeatable. I'll add a comment.